src/code_80086E70.c

```diff
@@ -580,226 +580,344 @@ void func_800887C0(s32 objectIndex) {
     set_object_flag_unk_054_false(objectIndex, 0x00800000);
 }
 
-UNUSED s32 func_80088880(s32 objectIndex, Player *player) {
-    s32 temp_f10;
-    s32 temp_f6;
-
-    temp_f10 = gObjectList[objectIndex].pos[0] - player->pos[0];
-    temp_f6  = gObjectList[objectIndex].pos[2] - player->pos[2];
-    return (temp_f10 * temp_f10) + (temp_f6 * temp_f6);
-}
-
-UNUSED s32 func_800888E8(s32 objectIndex, Player *player) {
-    s32 temp_f10;
-    s32 temp_f18;
-    s32 temp_f6;
-
-    temp_f10 = gObjectList[objectIndex].pos[0] - player->pos[0];
-    temp_f6  = gObjectList[objectIndex].pos[1] - player->pos[1];
-    temp_f18 = gObjectList[objectIndex].pos[2] - player->pos[2];
-    return (temp_f10 * temp_f10) + (temp_f6 * temp_f6) + (temp_f18 * temp_f18);
-}
-
-u32 func_8008896C(s32 objectIndex, Camera *camera) {
-    s32 temp_f10;
-    s32 temp_f6;
-
-    temp_f10 = gObjectList[objectIndex].pos[0] - camera->pos[0];
-    temp_f6  = gObjectList[objectIndex].pos[2] - camera->pos[2];
-    return (temp_f10 * temp_f10) + (temp_f6 * temp_f6);
-}
-
-UNUSED s32 func_800889D4(s32 objectIndex, Camera *camera) {
-    s32 temp_f10;
-    s32 temp_f18;
-    s32 temp_f6;
-
-    temp_f10 = gObjectList[objectIndex].pos[0] - camera->pos[0];
-    temp_f6  = gObjectList[objectIndex].pos[1] - camera->pos[1];
-    temp_f18 = gObjectList[objectIndex].pos[2] - camera->pos[2];
-    return (temp_f10 * temp_f10) + (temp_f6 * temp_f6) + (temp_f18 * temp_f18);
-}
-
-s32 func_80088A58(s32 objectIndex, Player *player, f32 arg2) {
-    f32 temp_f0;
-    f32 temp_f2;
-    s32 var_v1;
-
-    var_v1 = 0;
-    temp_f0 = gObjectList[objectIndex].pos[0] - player->pos[0];
-    temp_f2 = gObjectList[objectIndex].pos[2] - player->pos[2];
-    if (((temp_f0 * temp_f0) + (temp_f2 * temp_f2)) <= (arg2 * arg2)) {
-        var_v1 = 1;
+/**
+ * @brief Returns the distance between the object and the player.
+ * 
+ * @param objectIndex 
+ * @param player 
+ * @return UNUSED 
+ */
+UNUSED s32 distance_object_player_2d(s32 objectIndex, Player *player) {
+    s32 x;
+    s32 y;
+
+    x = gObjectList[objectIndex].pos[0] - player->pos[0];
+    y  = gObjectList[objectIndex].pos[2] - player->pos[2];
+    return (x * x) + (y * y);
+}
+
+/**
+ * @brief Returns the distance between the object and the player.
+ * 
+ * @param objectIndex 
+ * @param player 
+ * @return UNUSED 
+ */
+UNUSED s32 distance_object_player_3d(s32 objectIndex, Player *player) {
```

Comment:
get_distance_from_object_to_player
or
get_distance_to_player
I don't think 3d is necessary